services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/FacetModel.java

```diff
@@ -39,7 +39,7 @@
 	 * @param sqlQuery the sqlQuery on which to base the generated facet queries.
 	 * @param returnFacets whether facet information will be returned back to the user
 	 */
-	public FacetModel(List<FacetColumnRequest> selectedFacets, SqlQuery sqlQuery, boolean returnFacets) {
+	public FacetModel(List<FacetColumnRequest> selectedFacets, SqlQuery sqlQuery, boolean returnFacets, Long userId) {
```

Comment:
I think you can use the new SqlQuery userId variable (you can add a getter to the SqlQuery class) without having to pass it down the chain